pkg/util/cmd/cmd.go

```diff
@@ -32,6 +33,9 @@ var registerEncoders sync.Once
 func RunRootCommand(rootCmd *cobra.Command) {
 	registerEncoders.Do(func() {
 		zap.RegisterEncoder("tidb", func(cfg zapcore.EncoderConfig) (zapcore.Encoder, error) {
+			return log.NewTextEncoder(&log.Config{})
```

Comment:
Have a manual test to verify that the log configurations in the configuration file can work.

Reply:
`LogLevel` has no relation with `Encoder`. If we want to apply all zap config, either we need to modify `pingcap/log`, or we need to copy the whole file.

`LogFile` support is not added. If I rememered it correct, it is not added in the original weir, too. I can add it to my TODO list. This should be a trivial change.

Reply:
`LogLevel` tested, `LogFile` not working expectedly.

Reply:
File an issue to support `LogFile`.

Reply:
> File an issue to support `LogFile`.

It is recorded in the list already.